### PR TITLE
specify navtitle in the page metadata

### PR DESCRIPTION
**Pull request by afeld**
This means we can go back to using `./go update_nav`, rather than needing to keep the `navigation` in `_config.yml` up-to-date manually. Mostly wanted to confirm that https://github.com/18F/guides-style/pull/32 works, which it does!

/cc @mbland

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,7 +4,8 @@ gem 'jekyll'
 gem 'go_script'
 
 group :jekyll_plugins do
-  gem 'guides_style_18f'
+  # https://github.com/18F/guides-style/pull/32
+  gem 'guides_style_18f', github: '18F/guides-style', branch: 'navtitle'
```
Comment (afeld) [inline Gemfile]: This is meant to be temporary – we can wait to merge this until that one is done and a new release is cut if folks prefer.

**Comment by DavidEBest** [comment]
Might as well wait until that is merged in. Looks close to being done. :rocket:

**Comment by afeld** [comment]
@mbland Seems like the go_script gem upgrade may be causing problems...

https://github.com/18F/go_script/compare/v0.1.5...v0.1.6?w=1

Would you recommend running `./go ...` without `bundle exec`?

**Comment by mbland** [comment]
@afeld Yeah, you shouldn't need `bundle exec` before `./go` anymore. Also, looks like you've got some merge conflicts.

**Comment by afeld** [comment]
Hmmmm, still no luck. Should I be listing `bundler` in the Gemfile?

**Comment by afeld** [comment]
Downgraded go_script...will deal with that separately.

**Pull request merged**